specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,650 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Microsoft Defender for Cloud",
+		"description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+		"version": "2023-07-01-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security": [
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/{scope}/providers/Microsoft.Security/integrations": {
+			"get": {
+				"x-ms-examples": {
+					"List integrations by subscription scope": {
+						"$ref": "./examples/integrations/ListBySubscriptionIntegrations_example.json"
+					},
+					"List integrations by security connector scope": {
+						"$ref": "./examples/integrations/ListBySecurityConnectorIntegrations_example.json"
+					},
+					"List integrations by management group scope": {
+						"$ref": "./examples/integrations/ListByManagementGroupIntegrations_example.json"
+					}
+				},
+				"tags": [
+					"Integrations"
+				],
+				"description": "Get a list of all relevant integrations over a scope",
+				"operationId": "Integrations_List",
+				"parameters": [
+					{
+						"$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+					},
+					{
+						"$ref": "#/parameters/Scope"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/IntegrationsList"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed",
+						"schema": {
+							"$ref": "../../../common/v1/types.json#/definitions/CloudError"
+						}
+					}
+				},
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		},
+		"/{scope}/providers/Microsoft.Security/integrations/metadata": {
```

Comment:
>metadata [](http://example.com/codeflow?start=54&length=8)

This resource is not modelled correctly according to ARM semantics. This will be interpreted as a resource-collection od "integrations with the resource-id of name "metadata" . More details here : [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#uniform-resource-locators-urls](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#uniform-resource-locators-urls)